IMLS_scraper/__init__.py

```diff
@@ -0,0 +1,29 @@
+import logging
+import json
+import os
+import sys
+
+import datetime
+from pymongo import MongoClient, TEXT
+import azure.functions as func
+
+from .imls_scaper import imls_scaper
+
+
+def main(mytimer: func.TimerRequest, context: func.Context) -> None:
+    utc_timestamp = datetime.datetime.utcnow().replace(
+        tzinfo=datetime.timezone.utc
+    ).isoformat()
+    conn_string = os.environ['MONGO_DB_CONNECTION_STRING']
+    client = MongoClient(conn_string)['shelter']
+    scraped_update_date = imls_scaper.scrape_updated_date()
```

Comment:
Will do

Reply:
I am thinking we should have it only in the scraper so that later if we move away from azure functions, all logic will be available in scraper itself. Also this allows scraper to run independently and init.py only works as azure function controller with no complex logic. I will write the issue to remove extra logic from other init files.